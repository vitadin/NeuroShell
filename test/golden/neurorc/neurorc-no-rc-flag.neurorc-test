# CLI flag --no-rc test
# Tests that --no-rc flag skips all .neurorc files

--- description
Test --no-rc CLI flag skips all .neurorc files

--- working_dir_neurorc
\echo This should NOT appear - working dir .neurorc
\set[should_not_run="true"]

--- home_dir_neurorc
\echo This should NOT appear - home dir .neurorc
\set[should_not_run_home="true"]

--- cli_flags
--no-rc

--- input_sequence
\echo Testing --no-rc flag
\exit

--- expected_contains
Testing --no-rc flag

--- expected_not_contains
This should NOT appear - working dir .neurorc
This should NOT appear - home dir .neurorc
Setting should_not_run = true
Setting should_not_run_home = true