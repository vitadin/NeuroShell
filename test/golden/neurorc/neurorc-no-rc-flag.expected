[K/test/pwd []
Testing --no-rc flag
[K/test/pwd []